src/bundler/mod.rs

```diff
@@ -138,6 +138,9 @@ impl Bundler {
         let tx_hash = tx.tx_hash();
         trace!("Send bundle with transaction: {tx:?}");
 
+        // wait some time, otherwise sometimes events for this bundle are not yet detected
+        thread::sleep(Duration::from_millis(25));
```

Comment:
use `tx.await` ?

Reply:
Yeah, that's better. I updates the polling interval for the pending transaction from 7 secs to 0.1, so 1 test (that contains sending bundle) is not running for 7 seconds +.